jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -146,7 +147,11 @@ public void buildDocker()
 
     } catch (AppRootInvalidException ex) {
       throw new GradleException(
-          "container.appRoot is not an absolute Unix-style path: " + ex.getInvalidAppRoot());
+          "container.appRoot is not an absolute Unix-style path: " + ex.getInvalidPathValue());
```

Comment:
Just noticed - should we include cause `ex` in these?

Reply:
Also, looked over `AppRootInvalidException` and it seems that in the constructor, `message` and `invalidAppRoot` are always set to the invalid `appRoot`.

Reply:
+1 to include cause `ex`.

It is actually intended that the message is set to `appRoot` (like `FileNotFoundException`, whose `message` is simply the missing file path).

Reply:
Yea, is the `message` parameter necessary then? I couldn't find a call to the constructor that wasn't `appRoot` repeated twice (`new AppRootInvalidException(appRoot, appRoot, ex)`).

Reply:
Not really necessary in our codebase, but I think it makes sense that, conceptually, the general exception message is set to `appRoot` rather than leaving it undefined. If anyone is doing a very general exception handling of displaying a message, it would tell the invalid value.